packages/components/src/components/hds/advanced-table/index.hbs

```diff
@@ -0,0 +1,107 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<div
+  class={{this.classNames}}
+  ...attributes
+  role="grid"
+  aria-describedby={{this._captionId}}
```

Comment:
@MelSumner heads up: I ended up using aria-describedby instead of aria-description for the advanced table caption because it needs to be aria-live="polite"